handlers/docker/docker_client.go

```diff
@@ -1,4 +1,4 @@
-package dockerClient
+package docker
```

Comment:
will add a new method to allow initialize client from both environment and base url.